pkg/util/locode/db/continents/geojson/calls.go

```diff
@@ -31,18 +31,33 @@ func (db *DB) PointContinent(point *locodedb.Point) (*locodedb.Continent, error)
 
 	planarPoint := orb.Point{point.Longitude(), point.Latitude()}
 
-	var continent string
+	var (
+		continent string
+		minDst    float64
+	)
 
 	for _, feature := range db.features {
 		if multiPolygon, ok := feature.Geometry.(orb.MultiPolygon); ok {
 			if planar.MultiPolygonContains(multiPolygon, planarPoint) {
 				continent = feature.Properties.MustString(continentProperty)
 				break
+			} else {
+				distance := planar.DistanceFrom(multiPolygon, planarPoint)
```

Comment:
I'd set an `orb.Geometry` variable in if-else block, and calculate distance outside the block identically.